news_items/001_new_page_layout/index.html

```diff
@@ -0,0 +1,29 @@
+<p>
+  The layout of instructor pages and navigation bars has been
+  massively improved by <a href="https://github.com/tbretl">Tim
+    Bretl</a>.
+</p>
+
+<p>
+  <b>Easy switching.</b> You can now quickly switch between different
+  courses and different instances (semesters/terms) of a course. To do
+  this, click on the downwards-pointing triangle to the right of the
+  course name or instance name.
+</p>
+
+<p>
+  <b>Shortcut tabs in the top menu bar.</b> All pages can be accessed
+  from the second-level menu bar, but for commonly used pages like
+  "Questions" and "Assessments" there are also shortcut tabs in the
+  top menu bar.
+</p>
+
```

Comment:
The language here is a little confusing, but perhaps only because I'm used to what you call things in html/bootstrap. 😄 

I might try to stay consistent about...
* Calling what is at the top of the page the "navigation bar" (not the "top menu bar" for example).
* Calling items in the navigation bar "buttons" or "items" or something that is not "tabs".
* Calling what is in the page the "tabbed menu bar" or something, which has "tabs".

Reply:
done